package-lock.json

```diff
@@ -1,8 +1,17809 @@
 {
   "name": "rrr-frontend",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This is good to get updated, but could you verify that it was generated from the latest on `main`?

Reply:
@mathcolo verified